apps/src/templates/projects/PersonalProjectsTableActionsCell.jsx

```diff
@@ -93,19 +93,20 @@ export class PersonalProjectsTableActionsCell extends Component {
         {isEditing && (
           <div>
             <Button
-              __useDeprecatedTag
               onClick={this.onSave}
-              color={Button.ButtonColor.brandSecondaryDefault}
               text={i18n.save()}
+              size="s"
+              color={buttonColors.purple}
```

Comment:
I think you can get rid of the `buttonColors` here. This is how I've been setting the color, and it works like this on Storybook.
```suggestion
              color="purple"
```

Reply:
🤔 `buttonColors.purple` and `"purple"` end up being equivalent, but I think we want to use `buttonColors` since it's easier to search and update later. For example if we ever want to change "purple" to something more semantic. The Buttons [story](https://github.com/code-dot-org/code-dot-org/blob/09601303feea4a6f2a2a2b55ab51dd551b5db39e/apps/src/componentLibrary/button/Button.story.tsx#L84) uses `buttonColors` and so do the majority of `Button` usages that I see with a quick search.

Reply:
Oh that's a great point! I was looking at the buttons on the Storybook site (not the code file) and it was using the `"purple"` by itself so I used that, but I'll make a note to start using `buttonColors`.